advanced/scipy_sparse/examples/lobpcg_sakurai.py

```diff
@@ -21,7 +21,7 @@ def sakurai(n):
     """
 
     A = sp.sparse.eye(n, n)
-    d0 = np.array(r_[5, 6 * ones(n - 2), 5])
+    d0 = np.r_[5, 6 * np.ones(n - 2), 5]
```

Comment:
This is pre-existing code, so no need to update, but I'd nowadays use:

```suggestion
    d0 = np.hstack([5, 6 * np.ones(n - 2), 5])
```